.github/workflows/complete-support-request-info.yml

```diff
@@ -0,0 +1,132 @@
+name: Support Request - Complete Support Request Info
+
+on:
+ issues:
+    types: [opened]
+
+env:
+  FORM_TYPE: form-type-support
```

Comment:
This may no longer be necessary.

Reply:
removed as I removed the label from the issue template